templates/test/overview.html.ep

```diff
@@ -11,10 +11,10 @@
     <div id="summary" class="panel <%= ($aggregated->{failed} + $aggregated->{incomplete}) ? 'panel-danger' : 'panel-success' %>">
         <div class="panel-heading">
             Overall Summary of
-            % if ($group) {
-                <b><%= link_to $group->name => url_for('group_overview', groupid => $group->id) %></b>
-            % } else
-            % {
+            % if (@$groups) {
+                <b><%= b join(', ', map { link_to $_->name => url_for('group_overview', groupid => $_->id) } @$groups) %></b>
```

Comment:
I don't know what the problem is. See the test, it's requesting "/tests/overview?distri=opensuse&version=13.1&groupid=1001&groupid=1002"
So based on whatever job groups and database content you have you can do the same with two job groups which share the same product and build, e.g. all children groups of a parent group should work.